scripts/covid_tracking_project/historic_state_data/covid_tracking_new_schema.mcf

```diff
@@ -0,0 +1,45 @@
+Node: dcid:CovidTrackingProject
```

Comment:
Can you put this node in the new file? and typeOf: StatisticSourceEnum

Reply:
Yep, actually I already put this into enum_instances.mcf file. I will make the corresponding change.